aiven/client/pretty.py

```diff
@@ -62,69 +89,127 @@ def format_item(key, value):
     return formatted
 
 
+class PreparedTableData:
+    """PreparedTableData represents prepared data for methods which render tables."""
+
+    def __init__(self, fields, table_rows, horizontal_fields, vertical_fields):
+        self._fields = fields
+        self._table_rows = table_rows
+        self._horizontal_fields = horizontal_fields
+        self._vertical_fields = vertical_fields
+
+    @property
+    def fields(self):
+        return self._fields
+
+    @property
+    def table_rows(self):
+        return self._table_rows
+
+    @property
+    def horizontal_fields(self):
+        return self._horizontal_fields
+
+    @property
+    def vertical_fields(self):
+        return self._vertical_fields
+
+
+def prepare_table(result, table_layout=None):
+    if result is None:
+        return PreparedTableData([], [], [], [])
+
+    if not isinstance(result, (Sequence, )) or isinstance(result, (basestring, )):
```

Comment:
There are many of these sorts of checks, might be good to refactor them out into some helper methods. Also not sure why a tuple containing one item is being passed to `isinstance`.

Reply:
AFAIK (it may have changed in more recent versions) `isinstance` has always wanted a tuple.

Reply:
Yep fair enough. It is not necessary to pass a tuple though so might as well just pass the type.

Reply:
Fixed.